system-configuration/src/network_configuration.rs

```diff
@@ -0,0 +1,452 @@
+// Copyright 2017 Amagicom AB.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+//! Bindings to [`SCNetworkConfiguration`].
+//!
+//! See the examples directory for examples how to use this module.
+//!
+//! [`SCNetworkConfiguration`]: https://developer.apple.com/documentation/systemconfiguration/scnetworkconfiguration
+
+use core_foundation::array::CFArray;
+use core_foundation::base::{CFType, TCFType};
+use core_foundation::base::kCFAllocatorDefault;
+use core_foundation::dictionary::CFDictionary;
+use core_foundation::string::{CFString, CFStringRef};
+
+use dynamic_store::SCDynamicStoreBuilder;
+pub use system_configuration_sys::network_configuration::*;
+use system_configuration_sys::preferences::SCPreferencesCreate;
+
+use std::{fmt, ptr};
+use std::net::IpAddr;
+
+/// MTU
+#[derive(Debug)]
+pub struct SCNetworkInterfaceMTU {
```

Comment:
@faern do you suggest we should leave the uppercase prefix but switch the rest to the CamelCase? I.e `SCNetworkServiceDns` instead of `SCNetworkServiceDNS`?

Reply:
Yes. I suggest we should keep the `SC` prefix to be consistent with `core-foundation` and that we should follow the standard in all other instances.

Reply:
Okay.